docs/references/development_workflow.rst

```diff
@@ -82,3 +84,78 @@ Within the learning equality repository:
 If a change needs to be introduced to an older release, target the oldest release branch that we want the change made in. Then that change will need to be merged into all subsequent releases, one-at-a-time, until it eventually gets back to ``develop``.
 
 
+Github Labels
+-------------
+
+We use a `wide range of labels <https://github.com/learningequality/kolibri/labels>`__ to help organize issues and pull requests in the Kolibri repo.
+
+Changelog
+~~~~~~~~~
+
+The **changelog** label is used on PRs or issues to generate 'more details' links in the :ref:`changelog`.
+
+
+PR Lifecycle
+~~~~~~~~~~~~
+
+These labels help coordinate code reviews and our pull-request workflow. A PR should always have exactly one of these labels:
+
+* **PR: needs review**
+* **PR: needs updates**
+* **PR: work-in-progress**
+
+
+Development Category
+~~~~~~~~~~~~~~~~~~~~
+
+Used to help organize issues and sometimes PRs by area of responsibility or scope of domain knowledge necessary. They are not mutually exclusive.
+
+* **DEV: backend**
+* **DEV: content renderers**
+* **DEV: content**
+* **DEV: dev-ops**
+* **DEV: distributions**
+* **DEV: frontend**
+* **DEV: performance**
+
+
+Priority
+~~~~~~~~
+
+These are used to sort issues and sometimes PRs by priority if *and only if* the item is assigned a milestone. Every issue in a milestone ought to have a priority label.
```

Comment:
I got the idea that you would rather that a release maintainer assigns priority but that anyone with access can suggest the milestone?

Reply:
The way we've been approaching it is that the product lead (@rtibbles) sets high-level goals about what should be accomplished in each milestone, and the product development manager (myself) sets short-term priorities within the milestone in order to achieve those goals and simultaneously address important bugs, tech debt, and UX improvements.

Both of these should happen based on input from the team, our users, and our contributors.

Reply:
One thing I was thinking about was adding another dimension: 'severity'

that way we have some metadata which is decoupled from release planning

Reply:
Although not very relevant to the labels, I think it's important that the milestone is okay for anyone to set when they feel it's obvious which release a bug affects and will be fixed in. Or a PR already targeting a specific branch.

Maybe add in the description that priority is usually assigned by a core member?

Reply:
> Maybe add in the description that priority is usually assigned by a core member?

will do

>  I think it's important that the milestone is okay for anyone to set when they feel it's obvious which release a bug affects and will be fixed in. Or a PR already targeting a specific branch.

Yeah for sure people should be setting milestones on the PRs and issues they're working on, or for clearly important or critical issues.

The main situation where we might need additional process is when adding work to a milestone that  might affecting timelines. In that case there should be a conversation to make sure we can have resources to work on it, or adjust date expectations if necessary.